### ISSUE TITLE
Support for local Markdown sources in Redoc

### ISSUE THREAD

**Issue by frdmn** [RocketChat/Rocket.Chat.Docs#12]
As discussed with @marceloschmidt, it would be way more comfortable for us to contribute to the docs, if Redoc would support local sources to search for Markdown files instead of the public GitHub repository requirement as of now.

**Comment by engelgabriel** [comment]
Indeed. But do you know that you can use the GitHub editor and preview and it works OK? Even referencing media assets on the current folder. We should work on making the MD parser become 100% compatible with GitHub so there will be no surprises.

**Comment by frdmn** [comment]
@engelgabriel I am aware of the preview editor. I just had the feeling that, since we are pushing to the productive docs system, perhaps a local test would be more thorough to prevent unexpected behaviour even after the processing of the Markdown formatting. I thought about possible JS/CSS modifications that render the markup/images/etc. differently.

**Comment by engelgabriel** [comment]
You are right, I am just giving a temporary workaround.

We could also deploy a develop branch of the documentation into a different URL, and update that automatically on PRs.

**Comment by frdmn** [comment]
Yeah, I am totally fine with the temporary workaround, was just putting out my thoughts. :grin: 

Good idea with the secondary branch, but I would rather update on merge into the develop branch because otherwise a random person could raise a PR with malicious or unwanted piece of contribution and it gets automatically deployed on the development docs system.

**Comment by engelgabriel** [comment]
Agreed, I meant to say on merges, like our main project.

**Comment by frdmn** [comment]
Hehe, I just realized :+1:
